fcd/llvm-gvn-rewrite/MemorySSA.h

```diff
@@ -362,7 +362,7 @@ void setIncomingValue(unsigned int v, MemoryAccess *MA) {
 	if (Val.second != MA) {
 		if (Val.second) {
 			bool existsElsewhere = false;
-			for (unsigned i = 0, e = Operands.size(); i != e; ++i) {
+			for (unsigned i = 0, e = (unsigned int)Operands.size(); i != e; ++i) {
```

Comment:
I'm going to replace that file with an up-to-date one very soon, this will be overwritten. Until MemorySSA is distributed in a stable release, I think that it's better to level down warnings to the same level as LLVM builds with for this specific file.